riot/lol/match.go

```diff
@@ -14,6 +14,35 @@ type MatchClient struct {
 	c *internal.Client
 }
 
+// MatchListOptions
+type MatchListOptions struct {
+	// Set of champion IDs for filtering the matchlist.
+	Champion int
```

Comment:
It says that it's a SET of IDs, however it's only an `int`.
```suggestion
	Champion []int
```